rotkehlchen/db/updates.py

```diff
@@ -59,7 +60,7 @@ class RotkiDataUpdater:
     def __init__(self, msg_aggregator: 'MessagesAggregator', user_db: 'DBHandler') -> None:
         self.msg_aggregator = msg_aggregator
         self.user_db = user_db
-        self.branch = 'develop'
+        self.branch = os.getenv('GITHUB_BASE_REF', 'develop')
```

Comment:
Please don't mix PRs. EVER. I have also repeated this many times. This is a fix that should be on its own PR. You are blocking bugfixes merges by mixing it in with another PR.